srml/metadata/src/lib.rs

```diff
@@ -325,7 +328,7 @@ impl Decode for RuntimeMetadataDeprecated {
 /// The metadata of a runtime version 1.
 #[derive(Eq, Encode, PartialEq)]
 #[cfg_attr(feature = "std", derive(Decode, Debug, Serialize))]
```

Comment:
The documentation for this type is not correct and needs to be updated.

Reply:
Fixed on my PR